js-intermediate.md

```diff
@@ -5,102 +5,94 @@ Answer the following questions. First, without external resources. Challenge you
+  Researched answer:
 
 
 4. What is the difference between a class and an object?
 
-  Your answer: A class is a template for which objects can exist in. and object is the main object/ varialbe of a class.
+  Your answer:
 
-  Researched answer: The difference is simple and conceptual. A class is a template for objects. ... An object is a member or an "instance" of a class. An object has a state in which all of its properties have values that you either explicitly define or that are defined by default settings.
+  Researched answer:
 
 
 5. What did you learn during the group project this week? Please include any additional feedback you may have.
 
-  Your answer:People in your group may not have the same approach to the solution or way to tackle a task and that can be frustrating. After trying out their way and learing why it does or doesnt work is actually beneficial to you in the long run, regardless if it wasnt the way you wanted to do it. Also React is primarily used for bragging rights.
+  Your answer:
 
 
 
 ### HTML/CSS Review questions: First, try to answer each question on your own then Google the answer to further your knowledge.
 
 1. How do you link a CSS file to your HTML page?
 
-  Your answer:import /.App.css at the top of your html page
+  Your answer:
 
-  Researched answer:The external Style Sheet (.css file) always seperate from HTML file. You can link this external file (.css file) to your HTML document file using the < link > tag . You can place this < link > tag Within the < head > section, and after the < title > element of your HTML file.
+  Researched answer:
 
 
 2. What is the difference between a div and a span?
 
-  Your answer: im not entirely sure what a span is. but a div is a line-break used to group code.
+  Your answer:
 
-  Researched answer:The span tag is used with inline elements whilst the div tag is used with block-level content.
+  Researched answer:
 
 
 3. What is a CSS class? When should you use an id instead of a class?
 
-  Your answer:You call on your defined class to style it in your css component by using period "classname". You would use Id when you only want your css to apply to 1 variable because ID is for a certain variable while class can be used more than once.
+  Your answer:
 
-  Researched answer: The .class selector selects elements with a specific class attribute. To select elements with a specific class, write a period (.) character, followed by the name of the class. You can also specify that only specific HTML elements should be affected by a class.
-  
-  A class can be used several times, while an ID can only be used once, so you should use classes for items that you know you're going to use a lot.
+  Researched answer:
 
 
 4. Name 4 semantic HTML tags.
 
-  Your answer:not sure what these are.
+  Your answer:
 
```

Comment:
You found examples of semantic tags in the researched section, but to expand on the concept: Older html tags tend to be abbreviated. This became problematic for accessibility devises. The computer can read the tag along with the content for a more descriptive experience. Semantic tags are just tags that are named for how they render content.